### PR TITLE
add integ test shell for integration with infra

### PR DESCRIPTION
**Pull request by tianleh**
### Description
add a shell for integ test to be integrated with infra CI.
 
### Issues Resolved
https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/3
 
### Check List
- [Will verify together with infra team ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by Tengda-He — APPROVED** [review]

File: integtest.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  PASSWORD=`echo $CREDENTIAL | awk -F ':' '{print $2}'`
+fi
+
+git clone https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
+
+cd opensearch-dashboards-functional-test
+
+npm install
+
+if [ $SECURITY_ENABLED ]
```
Comment (peterzhuamazon) [inline integtest.sh:82]: Not sure if this works as the value is not bool, but string, better just compare string with `=`

File: integtest.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  CREDENTIAL="admin:admin"
+  USERNAME=`echo $CREDENTIAL | awk -F ':' '{print $1}'`
+  PASSWORD=`echo $CREDENTIAL | awk -F ':' '{print $2}'`
+fi
+
+git clone https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
+
+cd opensearch-dashboards-functional-test
+
+npm install
```
Comment (peterzhuamazon) [inline integtest.sh:80]: Remove these

**Comment by peterzhuamazon** [comment]
Related to: https://github.com/opensearch-project/opensearch-build/issues/604

File: integtest.sh (hunk 1 above)
Comment (tianleh) [reply integtest.sh]: Good catch. Just fixed.

File: integtest.sh (hunk 2 above)
Comment (tianleh) [reply integtest.sh:80]: Sure. Sounds like infra logic will take care of this. Let me remove these.

**Review by seraphjiang — APPROVED** [review]
LGTM

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**